sopa/io/reader/wsi.py

```diff
@@ -48,15 +49,14 @@ def wsi(
         images[f"scale{key}"] = scale_image
 
     multiscale_image = MultiscaleSpatialImage.from_dict(images)
-    multiscale_image.attrs["metadata"] = tiff_metadata
+    sdata = SpatialData(images={image_name: multiscale_image})
```

Comment:
When `as_image is True`, we will have the transformation already set inside the attrs of the image (see below). This might lead to some issues, but we can probably keep it like this for now

Reply:
I am not sure I understand this comment here :|

Reply:
Don't worry it's a very minor thing